packages/mobile/src/account/actions.ts

```diff
@@ -26,6 +26,7 @@ export enum Actions {
   SET_RETRY_VERIFICATION_WITH_FORNO = 'ACCOUNT/SET_RETRY_VERIFICATION_WITH_FORNO',
   ACCEPT_TERMS = 'ACCOUNT/ACCEPT_TERMS',
   MIGRATE_ACCOUNT_BIP39 = 'MIGRATE_ACCOUNT_BIP39',
+  CLEAR_STORED_ACCOUNT = 'CLEAR_STORED_ACCOUNT',
```

Comment:
This should be `ACCOUNT/CLEAR_STORED_ACCOUNT`
Ignore the missing `ACCOUNT/` above it, that's just @cmcewen setting a bad example ;)